content/industry/index/index.json

```diff
@@ -0,0 +1,27 @@
+{
+  "seo": {
+    "title": "SSW Consulting Services | Over 25 years of experience developing awesome solutions on top of Angular, React, Blazor, .NET, Azure, GitHub, DevOps (was VSTS/TFS), Power Apps, Power Automate, Power BI, SharePoint, Dynamics and SQL Server",
+    "canonical": "/products"
+  },
+  "title": "Industries we work with",
+  "subTitle": "Discover how SSW's innovative technology solutions have empowered both profit and non-profit organizations to navigate complex challenges and achieve their strategic goals.",
+  "industryList": [
+    {
+      "name": "Finance",
+      "url": "https://www.ssw.com.au/industry/finance",
+      "description": "We leverage our deep expertise in .NET, Dynamics 365, and Azure to develop customized software solutions that enhance financial management, risk mitigation, and regulatory compliance for the finance sector. Our partnership with Microsoft empowers us to align cutting-edge technology with your strategic business needs effectively.",
+      "logo": "/images/thumbs/ssw-finance-thumbnail.jpg"
+    },
+    {
+      "name": "Non-Profit",
+      "url": "https://www.ssw.com.au/industry/non-profit",
+      "description": "Leveraging more than 30 years of technological expertise, we provide non-profit organizations with customized software solutions that cater specifically to their unique needs. Our experience enables us to enhance data management, donor relations, and fundraising efforts through tailored donor management systems, volunteer portals, fundraising platforms, regulatory compliance software, and CRM solutions. These tools are designed to be flexible, secure, and effective, helping non-profits navigate complex regulatory environments and maximize their impact."
+    },
+    {
+      "name": "Education",
+      "url": "https://www.ssw.com.au/industry/educational",
```

Comment:
change all urls to relative path to avoid refreshing the page, (i.e `/industry/educational`)